components/molecules/TopicViewerContent.tsx

```diff
@@ -24,8 +24,8 @@ const useStyles = makeStyles((theme) => ({
     "& > *": {
       /* NOTE: 各動画プレイヤーのレスポンシブ対応により、高さはpaddingTopによってwidthのpercentage分
        * 確保されるため、heightによる制限ではなくwidthによる制限をおこなう必要がある */
-      // NOTE: 4:3前提なので、16:9では40vhよりも狭い高さになる
-      maxWidth: "calc(40vh * 4 / 3)",
+      // NOTE: 4:3前提になっているが本当はアスペクト比に応じて最大高さを変えたい
+      maxWidth: "calc(50vh * 4 / 3)",
```

Comment:
@horimasumi 今のアプローチで高さ制限をゆるめていくと、解説部分の領域の確保が充分とはいえない状態になりそうでしたので、16:9基準のリサイズに戻しまして、4:3の動画の際はより縦幅をとることを許容するように変更しました。

しばらくしたのちに改めてStorybookにて確認をお願いいたします :pray:

```suggestion
      // NOTE: 16:9前提になっているが本当はアスペクト比に応じて最大高さを変えたい
      maxWidth: "calc(40vh * 16 / 9)",
```


こちらのような見え方になります

![image](https://user-images.githubusercontent.com/9744580/113275259-8c2a9b00-9319-11eb-8d94-ea885ff165e4.png)